### DISCUSSION TITLE
Cannot import Nook DRM Epubs

### DISCUSSION THREAD

**Discussion by rjemmons** [apprenticeharper/DeDRM_tools#1489] [Q&A]
I cannot import and remove the DRM from Nook Epubs with Calibre and the DeDRM plugin. I am able to import Kindle books with no problem. The DeDRM plugin removes the Kindle DRM.

Versions:

Calibre v 5.9.0
DeDRM v 7.0.3
Nook App for PC 1.11.0.4

Procedure:

I purchased a book on Barnes & Noble website. I installed the Nook App from the Windows store. Signed in with the Nook App. I could see the Book I purchased in my Nook Library. I clicked on the book and was able to read it in the Nook App. I located the Epub file at C:\Users\[MyUserName]\AppData\Local\Packages\BarnesNoble.Nook_ahnzqzva31enc\LocalState. I started Calibre with the  DeDRM installed. I drag the Nook Epub file into Calibre. The Epub book appears in my Calibre Library, but the DRM is not removed. I cannot read the book with the Calibre Book reader. I get the error message "This Book is locked by DRM ..."

How can I import Barnes & Noble Nook Epub books into Calibre, and remove the DRM?

**Comment by ElleKayEm** [comment]
The methods for retrieving the keys for B&N stopped working when they cut off the old desktop programs.  The key can be extracted from the Android app.  Here is a write-up on how:  https://www.aricrenzo.com/2019-12-13-Liberate-Your-Nook-Ebooks/

I don't know how much work has been done on the B&N part of beta DeDRM 7.0.3 so you might need to use calibre 4.23 and DeDRM 6.8.1 once you have your key.

**Reply by rjemmons** [reply]
Thanks for the info. Does the old Nook desktop program still work, and is there anywhere to download it from?

**Comment by ElleKayEm** [comment]
No, the old apps were completely cut off from communication with B&N.
